bin/publish-stories.js

```diff
@@ -0,0 +1,65 @@
+const request = require('request');
```

Comment:
I'd recommend using superagent as it returns promises and has a plugin based architecture